### PR TITLE
[ebml] update to 1.4.4 and enable UWP builds

### PR DESCRIPTION
**Pull request by robUx4**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by robUx4** [comment]
@microsoft-github-policy-service agree

File: ports/ebml/vcpkg.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
 {
   "name": "ebml",
-  "version": "1.4.2",
-  "port-version": 1,
+  "version": "1.4.4",
+  "port-version": 2,
```
Comment (jimwang118) [inline ports/ebml/vcpkg.json:4]:
```suggestion
```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Please run the command: `vcpkg x-add-version portname`, and then submit the running result.

File: ports/ebml/vcpkg.json (hunk 1 above)
Comment (robUx4) [reply ports/ebml/vcpkg.json:4]: Done, thanks for the tip :)

Comment (robUx4) [reply ports/ebml/vcpkg.json]: Sorry I misunderstood your original comment. I did remove the version to revert back to port-revision 0 for this new 1.4.4 version.

**Review by jimwang118 — APPROVED** [review]

**Comment by jimwang118** [comment]
The usage has been tested successfully locally.
```
find_package(EBML CONFIG REQUIRED)
target_link_libraries(main PRIVATE EBML::ebml)
```

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**